stories/1-Button.stories.js

```diff
@@ -13,3 +13,11 @@ export const primary = () => (
 );
 
 export const secondary = () => <Button onClick={action("clicked")}>Cancel</Button>;
+
+export const PrimaryWithText = () => (
+  <div>
+    <h2 style={{fontFamily: "Roboto", color:"grey", fontSize: "1.0rem"}}>Set goals directly on Github</h2>
+    <Button style={{width: "90%"}} primary>View Your Data</Button>
```

Comment:
The preference here would be re-usability. Can the inline style be moved to a `<ButtonCardStyle />`?

Reply:
actually disregard. It is not worth addressing here. 

I can address in my future implementation PR.

Reply:
I actually thought about that today. I can probably address that if you'd like me to 👍  Just let me know